src/main/java/uk/gov/companieshouse/overseasentitiesapi/OverseasEntitiesApiApplication.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.overseasentitiesapi;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import javax.annotation.PostConstruct;
+import java.util.TimeZone;
+
+@SpringBootApplication
+public class OverseasEntitiesApiApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(OverseasEntitiesApiApplication.class, args);
+    }
+
+    @PostConstruct
+    public void init() {
```

Comment:
Do we need this method now?

Reply:
strictly speaking no, but figured if I removed it then it would be forgotten about and we'll end up having date/time values being out of sync between mongo and the api, which might cause us time to investigate and resolve.

Reply:
Awesome, thanks for the update